maint/gen_abi.py

```diff
@@ -43,12 +43,12 @@ def gen_mpi_abi_internal_h(out):
             elif RE.match(r'\s*int MPI_(SOURCE|TAG|ERROR);', line):
                 # no need to rename status fields
                 out.append(line.rstrip())
-            elif RE.match(r'\s*int (reserved|internal)\[(\d+)\];', line):
+            elif RE.match(r'\s*int MPI_(reserved|internal)\[(\d+)\];', line):
```

Comment:
Maybe `r'\s*int .*(reserved|internal)\[(\d+)\];`?

Reply:
Sure thing. However, I'll hold a little this PR until @jeffhammond finishes his alphabetical sorting quest :wink: